bee-ledger/src/workers/pruning/error.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::{prelude::MilestoneIndex, MessageId};
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Minimum: {minimum}, Found: {found}")]
```

Comment:
Everyone's favourite question -- what's the convention here, are the capitals fine to use? Same with the rest of the errors here.

Reply:
true, it should be no leading capitalization and no dot at the end.

Reply:
fixed